### PR TITLE
Simplified docker usage.

### PR DESCRIPTION
**Pull request by dartartem**

### CODE REVIEW

File: buildSrc/src/main/groovy/WaitForMySql.java [hunk 1]
```diff
@@ -45,7 +45,7 @@ private void waitForConnection() {
       Connection connection = null;
       try {
 
-        String datasourceUrl = getenv("SPRING_DATASOURCE_URL", () -> String.format("jdbc:mysql://%s/eventuate?useSSL=true", getenv("DOCKER_HOST_IP", "localhost")));
+        String datasourceUrl = getenv("SPRING_DATASOURCE_URL", () -> String.format("jdbc:mysql://%s/eventuate?useSSL=true", "localhost"));
```
Comment (cer) [inline buildSrc/src/main/groovy/WaitForMySql.java:48]: Why use format?

File: customer-service/src/main/resources/application.properties [hunk 2]
```diff
@@ -5,10 +5,16 @@ logging.level.org.hibernate.SQL=DEBUG
 logging.level.org.springframework.retry=TRACE
 
 
-spring.datasource.url=jdbc:mysql://${DOCKER_HOST_IP:localhost}/eventuate
+spring.datasource.url=jdbc:mysql://localhost/eventuate
```
Comment (cer) [inline customer-service/src/main/resources/application.properties:8]: There is no point in making this change.

Comment (dartartem) [reply customer-service/src/main/resources/application.properties:8]: Why? I removed every mention of DOCKER_HOST_IP. Why leave it here?

File: buildSrc/src/main/groovy/WaitForMySql.java (hunk 1 above)
Comment (dartartem) [reply buildSrc/src/main/groovy/WaitForMySql.java:48]: missed it. I will fix.

File: customer-service/src/main/resources/application.properties (hunk 2 above)
Comment (cer) [reply customer-service/src/main/resources/application.properties:8]: It's not a big deal but it leaves open the option of using DOCKER_HOST_IP in some unusual situation.

Comment (dartartem) [reply customer-service/src/main/resources/application.properties:8]: Should I replace localhost by ${DOCKER_HOST_IP:localhost} everywhere in each project?

**Pull request merged**